src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -148,36 +148,10 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 		// Load tab setting configs
 		this._tabSettingConfigs = this.dashboardService.getSettings<Array<TabSettingConfig>>([this.context, 'tabs'].join('.'));
 
-		const pinnedDashboardTabs: IDashboardTab[] = [];
-		const alwaysShowTabs = allTabs.filter(tab => tab.alwaysShow);
-
-		this._tabSettingConfigs.forEach(config => {
-			if (config.tabId && types.isBoolean(config.isPinned)) {
-				const tab = find(allTabs, i => i.id === config.tabId);
-				if (tab) {
-					if (config.isPinned) {
-						pinnedDashboardTabs.push(tab);
-					} else {
-						// overwrite always show if specify in user settings
-						const index = firstIndex(alwaysShowTabs, i => i.id === tab.id);
-						alwaysShowTabs.splice(index, 1);
-					}
-				}
-			}
-		});
+		this.loadNewTabs(allTabs);
 
-		this.loadNewTabs(pinnedDashboardTabs);
-		this.loadNewTabs(alwaysShowTabs);
+		this.panelActions = [];
 
-		// Set panel actions
-		const openedTabs = [...pinnedDashboardTabs, ...alwaysShowTabs];
-		if (extensionTabsEnabled) {
-			const addNewTabAction = this.instantiationService.createInstance(AddFeatureTabAction, allTabs, openedTabs, this.dashboardService.getUnderlyingUri());
-			this._tabsDispose.push(addNewTabAction);
-			this.panelActions = [addNewTabAction];
-		} else {
-			this.panelActions = [];
-		}
 		this._cd.detectChanges();
 
 		this._tabsDispose.push(this.dashboardService.onPinUnpinTab(e => {
```

Comment:
Do you need this anymore if you're removing pinning/unpinning?

Reply:
pin/unpin might be managed differently, I might still need the code, new experience is still pending on the PM and UX.